Source/EasyNetQ/DefaultRpc.cs

```diff
@@ -223,6 +221,7 @@ CancellationToken cancellationToken
             string correlationId,
             TimeSpan expiration,
             byte? priority,
+            bool mandatory,
```

Comment:
@Pliner Just a note. This is actually a backward incompatible change. I propose you to add API approval tests to EasyNetQ to guard from such accidental API chages. See example [here](https://github.com/graphql-dotnet/server/tree/master/tests/ApiApprovalTests)

Reply:
Yep, you are right. Sad news.

By the way, do you extend RPC in that way(with an override of protected methods)?

Reply:
I am still using EasyNetQ version 3.x.x. I can confidently say that I have a lot of overrided methods (protected or public). I am going to migrate to the latest EasyNetQ version sometime in the future.